### ISSUE TITLE
Bundle update fails when coming from bundlelist

### ISSUE THREAD

**Issue by jonashoechst** [umr-ds/pyserval#17]
When a bundle is read by using get_bundlelist(), update_manifest() is not successful. 

```
Traceback (most recent call last):
  File "/serval_routing/helpers/rhizome-manifest-bug.py", line 17, in <module>
    list_bundle.update_manifest(test="value")
  File "/serval_routing/helpers/pyserval/rhizome.py", line 114, in update_manifest
    self.update()
  File "/serval_routing/helpers/pyserval/rhizome.py", line 83, in update
    bundle_secret=self.bundle_secret
  File "/serval_routing/helpers/pyserval/rhizome.py", line 416, in insert
    assert isinstance(payload, basestring) or isinstance(payload, bytes)
AssertionError
```

**Comment by CryptoCopter** [comment]
Actually expected behaviour, but probably lacking proper error reporting.

If you got the bundle from get_bundlelist, thenn its payload has not been loaded and so calling update would fail (best case) or delete the payload in the store (worst case)

**Comment by jonashoechst** [comment]
I'd propose a refresh method to reload a (full) bundle. Will be useful for this case, as well as bundles which are updated regularly.

**Comment by CryptoCopter** [comment]
Fixed in c8f60b6a2a47f1ae70d6949e1d61479341aa7204
